2022-11-01 20:32:45,578 INFO    MainThread:82121 [wandb_setup.py:_flush():68] Configure stats pid to 82121
2022-11-01 20:32:45,578 INFO    MainThread:82121 [wandb_setup.py:_flush():68] Loading settings from /Users/moeezmalik/.config/wandb/settings
2022-11-01 20:32:45,578 INFO    MainThread:82121 [wandb_setup.py:_flush():68] Loading settings from /Users/moeezmalik/Documents/Main/work/fraunhofer/thesis/Lightning-Table/wandb/settings
2022-11-01 20:32:45,579 INFO    MainThread:82121 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-01 20:32:45,579 INFO    MainThread:82121 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'network/trainers.py', 'program': 'network/trainers.py'}
2022-11-01 20:32:45,579 INFO    MainThread:82121 [wandb_init.py:_log_setup():476] Logging user logs to ./wandb/run-20221101_203245-2hnqowam/logs/debug.log
2022-11-01 20:32:45,579 INFO    MainThread:82121 [wandb_init.py:_log_setup():477] Logging internal logs to ./wandb/run-20221101_203245-2hnqowam/logs/debug-internal.log
2022-11-01 20:32:45,579 INFO    MainThread:82121 [wandb_init.py:init():516] calling init triggers
2022-11-01 20:32:45,579 INFO    MainThread:82121 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-01 20:32:45,580 INFO    MainThread:82121 [wandb_init.py:init():569] starting backend
2022-11-01 20:32:45,580 INFO    MainThread:82121 [wandb_init.py:init():573] setting up manager
2022-11-01 20:32:45,587 INFO    MainThread:82121 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-01 20:32:45,591 INFO    MainThread:82121 [wandb_init.py:init():580] backend started and connected
2022-11-01 20:32:45,597 INFO    MainThread:82121 [wandb_init.py:init():658] updated telemetry
2022-11-01 20:32:45,627 INFO    MainThread:82121 [wandb_init.py:init():693] communicating run to backend with 30 second timeout
2022-11-01 20:33:15,743 ERROR   MainThread:82121 [wandb_init.py:init():703] backend process timed out
2022-11-01 20:33:15,744 ERROR   MainThread:82121 [wandb_init.py:init():712] encountered error: Error communicating with wandb process
For more info see: https://docs.wandb.ai/library/init#init-start-error
